### ISSUE TITLE
corss entopy loss is different from hinge loss

### ISSUE THREAD

**Issue by maryamag85** [bfortuner/ml-cheatsheet#72]
confusing definition for cross-entropy loss 
referring to Stanford lecture notes 
http://cs231n.github.io/linear-classify/

you are calling log loss same as cross entropy loss

**Comment by xinbinhuang** [comment]
You can see log loss as a special case (with only 2 classes) of entropy loss.
